testing/test_doctest.py

```diff
@@ -227,7 +227,7 @@ def test_doctest_unex_importerror_with_module(self, testdir):
         # doctest is never executed because of error during hello.py collection
         result.stdout.fnmatch_lines([
             "*ERROR collecting hello.py*",
-            "*ImportError: No module named *asdals*",
+            "*Error: No module named *asdals*",
```

Comment:
I'm not sure @blueyed, it seems a fuzzy matching for the message here is enough. Do you see a situation where we obtain a match but it is the wrong match? Seems unlikely to me.

But I'm not entirely against the idea either, if @hroncok agrees this can easily be changed to:

```python
expected_exc = 'ModuleNotFoundError' if sys.version_info[:2] >= (3, 6) else 'ImportError'
result.stdout.fnmatch_lines([
     "*ERROR collecting hello.py*",
     "*%s: No module named *asdals*" % expected_exc,
```